packages/database/src/migrations/20210727041905-AddSyncCursorTable-modifies-schema.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`
+    CREATE TABLE sync_cursor (
```

Comment:
```suggestion
    CREATE TABLE sync_service (
```
Because it (rightly) contains more info than just the cursor, including config about what sync groups to sync, and in future maybe switches configuring how often it syncs etc.